EIPS/eip-2494.md

```diff
@@ -0,0 +1,363 @@
+---
+eip:      2494
+title:    Baby Jubjub Elliptic Curve
+authors:  Barry WhiteHat (@barryWhiteHat), Marta Bellés (@bellesmarta), Jordi Baylina (@jbaylina)
+discussions-to: https://ethereum-magicians.org/t/eip-2494-baby-jubjub-elliptic-curve/3968
+status:   Draft
+type:     Standards Track 
+category: ERC
+created:  2020-01-29
```

Comment:
```suggestion
eip: 2494
title: Baby Jubjub Elliptic Curve
author: Barry WhiteHat (@barryWhiteHat), Marta Bellés (@bellesmarta), Jordi Baylina (@jbaylina)
@MicahZoltu
MicahZoltu 7 minutes ago Collaborator
Suggested change
authors: Barry WhiteHat (@barryWhiteHat), Marta Bellés (@bellesmarta), Jordi Baylina (@jbaylina)
author: Barry WhiteHat (@barryWhiteHat), Marta Bellés (@bellesmarta), Jordi Baylina (@jbaylina)
@MicahZoltu
discussions-to: https://ethereum-magicians.org/t/eip-2494-baby-jubjub-elliptic-curve/3968
status: Draft
type: Standards Track 
category: ERC
created: 2020-01-29
```
The bot is very particular.  This isn't rendered as plain text anywhere other than when editing, so the manual formatting is unnecessary and makes automated updates to EIP metadata harder.